review/src/ReviewConfig.elm

```diff
@@ -29,8 +29,9 @@ config =
         |> Rule.ignoreErrorsForFiles [ "src/Views/Debug.elm" ]
       -- Common
     , NoExposingEverything.rule
-        |> Rule.ignoreErrorsForDirectories [ "tests/" ]
+        |> Rule.ignoreErrorsForFiles [ "src/Data/Color.elm" ]
```

Comment:
Why is this exception needed? Maybe a comment here?

Reply:
For exactly the same reason as `Icon.elm` is excluded: it's a catalog of items we want to fully expose, there's nothing "private" and above all we want to keep the ability of having "unused" colors in the module, because it's a colorset and it doesn't make sense to have it partially implemented.